modules/ROOT/pages/apikit-graphql-module-reference.adoc

```diff
@@ -0,0 +1,235 @@
+= APIkit for GraphQL Module Reference
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Configurations
+---
+
+=== Config
+
+Default configuration
+
+==== Parameters
+
+|===
+| Name | Type | Description | Default Value | Required
+| Name | String | The name for this configuration. Connectors reference the configuration with this name. | | *x*
+| API definition | String |  |  | *x*
+| Introspection Enabled | Boolean |  It enables schema introspection. |  false | 
+| Max Query Depth Allowed | Number |  The maximum depth a query can have. The value must be greater than 1. |  100 | 
+| Max Query Complexity Allowed | Number | The maximum number of data fields a query can include. The value must be greater than 1. |  100 | 
+| Name | String |  The identifier of this element is used to reference it in other components. |  | *x*
+|===
+
+==== Associated Operations
+
+* <<Route>> 
+* <<Serialize>> 
+* <<SerializeBatch>> 
+
+==== Associated Sources
+
+* <<DataFetcher>> 
+* <<DataLoader>> 
+
+== Operations
+
+=== Route
+
+`<graphql-router:route>`
+
+==== Parameters
+
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | *x*
+| Content | Any |  |  #[payload] | 
```

Comment:
This field has no description.

Reply:
"Content" is the content (usually the payload) of the message. See the [reference documentation](https://docs.mulesoft.com/mule-sdk/1.1/content-parameters#content-param) just in case you can thing on a different definition

Reply:
OK! Thanks!